packages/authentication/src/keys.ts

```diff
@@ -4,9 +4,10 @@
 // License text available at https://opensource.org/licenses/MIT
 
 import {Strategy} from 'passport';
-import {BindingKey} from '@loopback/context';
 import {AuthenticateFn, UserProfile} from './providers/authentication.provider';
 import {AuthenticationMetadata} from './decorators/authenticate.decorator';
+import {BindingKey, MetadataAccessor} from '@loopback/context';
```

Comment:
Shouldn't `MetadataAccessor` be imported from `@loopback/metadata`? Actually I'm not even sure how this is working since I didn't see a re-export of `MetadataAccessor` from `@loopback/context` and this class is defined in `@loopback/metadata`.